<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DelBookWindow</class>
 <widget class="QMainWindow" name="DelBookWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="rdb_bookid">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>图书号</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdb_bookname">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>图书名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="et_bookid">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="et_bookname">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delbook">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除图书</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
